File: MangaCount.Server/Model/MangaModel.cs
namespace MangaCount.Server.Model
{
    public class MangaModel
    {
        public int Id { get; set; }
        public required string Name { get; set; }
        public int? Volumes { get; set; }
        public int FormatId { get; set; }
        public int PublisherId { get; set; }
    }
}

File: mangacount.client/src/app/models/manga.model.ts
export interface Manga {
  id: number;
  name: string;
  volumes: number | null;
  formatId: number;
  format?: Format;
  publisherId: number;
  publisher?: Publisher;
}

export interface MangaCreateDto {
  id?: number;
  name: string;
  volumes: number | null;
  formatId: number;
  publisherId: number;
}

export interface Format {
  id: number;
  name: string;
}

export interface Publisher {
  id: number;
  name: string;
}
